src/rf/apps/workers/process.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from apps.core.models import LayerImage, LayerMeta
+from django.conf import settings
+
+import time
+import json
+import boto.sqs
+from boto.sqs.message import Message
+
+TIMEOUT_SECONDS = (60 * 10)  # 10 Minutes.
+STATUS_VALIDATED = 'validated'
+STATUS_REPROJECTED = 'reprojected'
+STATUS_PROCESSING = 'processing'
+STATUS_FAILED = 'failed'
+STATUS_COMPLETE = 'complete'
+
+JOB_VALIDATE = [
+    'ObjectCreated:CompleteMultipartUpload',
+    'ObjectCreated:Put',
+    'ObjectCreated:Post'
+]
+JOB_REPROJECT = 'reproject'
```

Comment:
I think this makes a good case for splitting jobs into separate classes. There's meta information like this that can be neatly grouped together and organized using OOP instead of loosely coupled variables.